src/en/tools-charm-tools.md

```diff
@@ -1,28 +1,38 @@
 Title: Charm tools
-TODO:  The top header should be "Charm Tools"
-       Review required
 
-The 'Charm Tools' Juju Plugin is a collection of commands enabling users and
-charm authors to create, search, fetch, update, and manage charms.
+# Charm Tools
+
+The Charm Tools is a collection of commands that assist with charm management.
+In particular, they allow charm authors to create charms.
 
 # Installation
 
-The source project can be found at
-[https://launchpad.net/charm-tools](https://launchpad.net/charm-tools).
+The Charm Tools can be installed on various platforms with the use of
+[Snaps][snaps]. The name of the snap is called `charm`.
 
 ## Ubuntu
 
-Install the `charm` snap:
+To install the Charm Tools on Ubuntu:
 
 ```bash
 sudo snap install charm
 ```
 
 ## Other Linuxes
 
-You can install Charm Tools on other Linux distributions using a snap too. See
-the upstream [Snaps documentation][snap-install] for getting snapd onto your
-Linux.
+To install on other Linux distributions see the upstream
+[Snaps documentation][snaps-install].
+
+## Non-Linux platforms
+
+It is feasible to install the Tools on non-Linux platforms such as macOS and
```

Comment:
I don't think this is a great idea. Perhaps simply better to redirect to the source project.

" There are no current builds of Charm Tools for other operating systems. Check the [Charm Tools project] for further information